### PR TITLE
Release v2.1.0-b1

### PR DESCRIPTION
**Pull request by HanlinMiao**
# Closes: N/A
# What's Changed

# TODO

- [ ] Explanation of Change(s)
- [ ] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments))
- [ ] Attached Screenshots, Payload Example
- [ ] Unit, Integration Tests
- [ ] Documentation Updates (when adding/changing features)
- [ ] Example Plugin Updates (when adding/changing features)
- [ ] Outline Remaining Work, Constraints from Design

### CODE REVIEW

File: nautobot/ui/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "nautobot",
-    "version": "2.1.0-b1",
+    "version": "2.1.0",
```
Comment (gsnider2195) [inline nautobot/ui/package.json:3]:
```suggestion
    "version": "2.1.0-b1",
```

Comment (gsnider2195) [reply nautobot/ui/package.json:3]: We'll bump the version after we cut the release. And we'll bump it to beta 2, not straight to 2.1.0

File: nautobot/docs/release-notes/version-2.1.md [hunk 2]
```diff
@@ -36,3 +36,51 @@ The data export functionality in all object list views (allowing export of all o
...
+- [#4796](https://github.com/nautobot/nautobot/issues/4796) - Added InterfaceRedundancyGroupAssociation to GraphQL.
+- [#4796](https://github.com/nautobot/nautobot/issues/4796) - Added IPAddressToInterface to GraphQL.
+- [#4796](https://github.com/nautobot/nautobot/issues/4796) - Added VRFDeviceAssignment to GraphQL.
+- [#4796](https://github.com/nautobot/nautobot/issues/4796) - Added VRFPrefixAssignment to GraphQL.
+- [#4820](https://github.com/nautobot/nautobot/issues/4820) - Added listing of related `files` to the `/api/extras/job-results/` REST API.
+- [#4820](https://github.com/nautobot/nautobot/issues/4820) - Added read-only REST API for the FileProxy model (files generated by a Job run), including a `/download/` endpoint for downloading the file content.
+
+### Changed
+
+- [#4677](https://github.com/nautobot/nautobot/issues/4677) - Updated and customized nautobot UI bootstrap theme with LESS variables.
```
Comment (glennmatthews) [inline nautobot/docs/release-notes/version-2.1.md:63]: This and 4765 deserve a mention in the Release Overview section.

File: nautobot/ui/package-lock.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
 {
     "name": "nautobot",
-    "version": "2.1.0-b1",
+    "version": "2.1.0",
```
Comment (glennmatthews) [inline nautobot/ui/package-lock.json:3]: revert changes to this file as well.

**Review by glennmatthews — APPROVED** [review]

**Pull request merged**